app/src/main/java/com/github/se/bootcamp/ui/screens/ChallengeScreen.kt

```diff
@@ -1,29 +1,202 @@
+            var isHelpBoxVisible by remember { mutableStateOf(false) }
+
+            // Main Column
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(Color.DarkGray)  // Whole background now white
+                    .padding(top = 32.dp), // Adjusted padding to move everything lower
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                // Top blue line
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(4.dp)
+                        .background(Color(0xFF05A9FB))
+                )
+
+                // Top row with Info button aligned to the right (blue button without border)
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(end = 16.dp),
+                    horizontalArrangement = Arrangement.End // Aligned to the right
+                ) {
+                    Button(
+                        onClick = { isHelpBoxVisible = !isHelpBoxVisible },
+                        colors = ButtonDefaults.buttonColors(containerColor = Color(0xFF05A9FB)), // Blue button without border
+                        modifier = Modifier.clip(RoundedCornerShape(12.dp)) // Ensure the shape matches exactly
+                    ) {
+                        Icon(Icons.Outlined.Info, tint = Color.White, contentDescription = "Help")
+                    }
+
+                    if (isHelpBoxVisible) {
+                        Dialog(onDismissRequest = { isHelpBoxVisible = false }) {
+                            Surface(
+                                shape = RoundedCornerShape(8.dp),
+                                color = Color.LightGray,
+                                modifier = Modifier
+                                    .padding(16.dp)
+                                    .border(2.dp, Color.Gray, RoundedCornerShape(12.dp))
+                                    .clip(RoundedCornerShape(12.dp))
+                            ) {
+                                Column(
+                                    modifier = Modifier
+                                        .padding(16.dp)
+                                        .fillMaxWidth(),
+                                    horizontalAlignment = Alignment.CenterHorizontally
+                                ) {
+                                    Text(
+                                        "This is your personal space where you can:\n" +
+                                                "\n" +
+                                                "- View and edit your profile.\n" +
+                                                "\n" +
+                                                "- Track your progress: See how far you have come in your ASL journey, with clear insights into what you have learned and what is next.",
```

Comment:
It's a good habit to develop to avoid writing hard coded string in the code, and favor the use of String.xml resources. This will makes it easier to later [localize](https://developer.android.com/guide/topics/resources/localization) the application. It also makes the app more maintainable as it is easy to modify one text, without having to remember to modify every places that are displaying the hard coded text.